### PR TITLE
Add The week in Montage for 2013-10-02

### PR DESCRIPTION
**Pull request by Stuk**

### CODE REVIEW

**Comment by Stuk** [comment]
@kriskowal I could especially use your help this week checking that my summary of collections/frb changes is correct.

File: _posts/2013-10-02-the-week-in-montage.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Continuing with the listener updates, the [`dispatchesRangeChanges` property on range-change dispatching objects is now non-enumerable](https://github.com/montagejs/collections/pull/34), which now includes the [List and Set](https://github.com/montagejs/collections/pull/33). In FRB the [Set now supports the `only` operator](https://github.com/montagejs/frb/pull/16), and after some [internal changes to the Scope object](https://github.com/montagejs/frb/pull/15) the [parent binder, `^`, has been added](https://github.com/montagejs/frb/pull/18).
```
Comment (kriskowal) [inline _posts/2013-10-02-the-week-in-montage.md]: A bug in range and map change listeners has been fixed such that `dispatchesRangeChanges` and `dispatchesMapChanges` have been made non-enumerable on observed arrays. This fixes an issue that made equivalent observed and non-observed arrays difficult to compare. Also, The List and Set collections can now dispatch range change events, which in turn makes them bindable through FRB expressions that could previously only react to changes in Array and SortedSet.

**Comment by kriskowal** [comment]
That is the only change I recommend.

**Comment by Stuk** [comment]
Thanks.

**Pull request merged**